src/app-layout/__tests__/main.test.tsx

```diff
@@ -115,6 +115,51 @@ describe.each([
   });
 });
 
+describe.each([
```

Comment:
We're passing a single array item to `describe.each(` here, so I think we can replace it with `describe(` and move the `find*` functions to standalone `const`s.